### PR TITLE
[zh] sync custom-resource-definitions define-environment-variable-container

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 1]
```diff
@@ -155,21 +155,23 @@ the list. Similarly, avoid circular references.
...
-container.
+Honorable`, and `Kubernetes`, respectively. The environment variable 
+`MESSAGE` combines the set of all these environment variables and then uses it 
+as a CLI argument passed to the `env-print-demo` container.
 -->
 ## 在配置中使用环境变量   {#using-env-var-inside-of-your-config}
 
-你在 Pod 的配置中定义的环境变量可以在配置的其他地方使用，
-例如可用在为 Pod 的容器设置的命令和参数中。
+你在 Pod 的配置中定义的环境变量，位于 `.spec.containers[*].env[*]`，
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container.md:169]:
```suggestion
你在 Pod 的配置中定义的、位于 `.spec.containers[*].env[*]` 下的环境变量
```

File: content/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 2]
```diff
@@ -155,21 +155,23 @@ the list. Similarly, avoid circular references.
...
+可以在配置的其他地方使用，例如可用在为 Pod 的容器设置的命令和参数中。
 在下面的示例配置中，环境变量 `GREETING`、`HONORIFIC` 和 `NAME` 分别设置为 `Warm greetings to`、
-`The Most Honorable` 和 `Kubernetes`。然后这些环境变量在传递给容器 `env-print-demo` 的 CLI 参数中使用。
+`The Most Honorable` 和 `Kubernetes`。 
+环境变量 `MESSAGE` 将所有这些环境变量的集合组合起来，
+然后在传递给容器 `env-print-demo` 的 CLI 参数中使用。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container.md:174]:
```suggestion
然后再传递给容器 `env-print-demo` 的 CLI 参数中使用。
```

**Review by windsonsea — COMMENTED** [review]
/approve

File: content/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 3]
```diff
@@ -155,21 +155,23 @@ the list. Similarly, avoid circular references.
...
 -->
 ## 在配置中使用环境变量   {#using-env-var-inside-of-your-config}
 
-你在 Pod 的配置中定义的环境变量可以在配置的其他地方使用，
-例如可用在为 Pod 的容器设置的命令和参数中。
+你在 Pod 的配置中定义的、位于 `.spec.containers[*].env[*]` 下的环境变量
+可以在配置的其他地方使用，例如可用在为 Pod 的容器设置的命令和参数中。
 在下面的示例配置中，环境变量 `GREETING`、`HONORIFIC` 和 `NAME` 分别设置为 `Warm greetings to`、
-`The Most Honorable` 和 `Kubernetes`。然后这些环境变量在传递给容器 `env-print-demo` 的 CLI 参数中使用。
+`The Most Honorable` 和 `Kubernetes`。
```
Comment (my-git9) [inline content/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container.md:172]:
```suggestion
`The Most Honorable` 和 `Kubernetes`。
```

**Pull request merged**